tests/puppeteer/test/linkchecker.js

```diff
@@ -127,7 +127,8 @@ const run = async () => {
 
   const curDate = new Date();
 
-  const filename = `report_${curDate.toJSON().slice(0, 10)}_${curDate.getHours()}${curDate.getMinutes()}${curDate.getSeconds()}.json`;
+  const filename = `report_${curDate.toJSON()
+    .slice(0, 10)}_${curDate.getHours()}${curDate.getMinutes()}${curDate.getSeconds()}.json`;
```

Comment:
Linter made this ?

Reply:
yes,
I add "max-len": [2, {"code": 120}] in linter and have appear . so going to new line make error disappear.

Reply:
better to present like this?
```js
const dateString = curDate.toJSON().slice(0, 10);
const hours = curDate.getHours();
const minutes = curDate.getMinutes();
const seconds = curDate.getSEconds();
const filename = `report_${dateString}_${hours}${minutes}${seconds}.json`;
```

Reply:
done thanks